specification/network/resource-manager/Microsoft.Network/stable/2022-09-01/azureFirewall.json

```diff
@@ -1148,6 +1210,127 @@
         }
       },
       "description": "List of SNAT IP Prefixes learnt by firewall to not SNAT"
+    },
+    "FirewallPacketCaptureParametersFormat": {
+      "type": "object",
+      "properties": {
+        "durationInSeconds": {
+          "type": "integer",
+          "format": "int32",
+          "maximum": 1800,
+          "exclusiveMaximum": false,
+          "minimum": 30,
+          "exclusiveMinimum": false,
+          "description": "Duration of packet capture in seconds."
+        },
+        "numberOfPacketsToCapture": {
+          "type": "integer",
+          "format": "int32",
+          "maximum": 90000,
+          "exclusiveMaximum": false,
+          "minimum": 100,
+          "exclusiveMinimum": false,
+          "description": "Number of packets to be captured."
+        },
+        "sasUrl": {
+          "type": "string",
+          "description": "Upload capture location"
```

Comment:
This should have x-ms-secret if it contains a sas url with secrets. [https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-secret](https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-secret)

Reply:
The SASURL does not contain secrets it will have only write create and add permissions to store packet capture on Cx storage account/container.